output/forms.eslint.txt

```diff
@@ -7,120 +7,32 @@ Scoping to packages that match 'react-talend-forms'
-  437:26  error  Do not nest ternary expressions                                                                no-nested-ternary
-  528:22  error  'onClick' is missing in props validation                                                       react/prop-types
-  528:31  error  'disabled' is missing in props validation                                                      react/prop-types
-  544:3   error  Prop type `object` is forbidden                                                                react/forbid-prop-types
-  552:3   error  Prop type `object` is forbidden                                                                react/forbid-prop-types
-  553:3   error  Prop type `object` is forbidden                                                                react/forbid-prop-types
-  556:3   error  Prop type `array` is forbidden                                                                 react/forbid-prop-types
+  195:32  error  Unexpected block statement surrounding arrow body                                              arrow-body-style
+  207:40  error  Unexpected block statement surrounding arrow body                                              arrow-body-style
+  225:32  error  Unexpected block statement surrounding arrow body                                              arrow-body-style
+  241:16  error  Expected 'this' to be used by class method 'isItemRequired'                                    class-methods-use-this
+  548:3   error  Prop type `object` is forbidden                                                                react/forbid-prop-types
+  556:3   error  Prop type `object` is forbidden                                                                react/forbid-prop-types
+  557:3   error  Prop type `object` is forbidden                                                                react/forbid-prop-types
+  560:3   error  Prop type `array` is forbidden                                                                 react/forbid-prop-types
+  566:3   error  Prop type `object` is forbidden                                                                react/forbid-prop-types
+  579:3   error  Prop type `object` is forbidden                                                                react/forbid-prop-types
+  586:3   error  Prop type `object` is forbidden                                                                react/forbid-prop-types
+  613:3   error  Prop type `object` is forbidden                                                                react/forbid-prop-types
+  614:3   error  Prop type `object` is forbidden                                                                react/forbid-prop-types
+  634:3   error  Prop type `object` is forbidden                                                                react/forbid-prop-types
+  635:3   error  Prop type `object` is forbidden                                                                react/forbid-prop-types
```

Comment:
Let's merge this one, and make another PR to update the eslintrc + remove the suppress `react/forbid-prop-types` in all files